esp-hal/src/dma/mod.rs

```diff
@@ -929,6 +948,16 @@ pub enum DmaExtMemBKSize {
     Size64 = 2,
 }
 
+impl From<DmaBufBlkSize> for DmaExtMemBKSize {
```

Comment:
Could these two enums just be the same? (Could be dealt with in a separate PR)

Reply:
One is the set of values for the DMA peripheral and the other is used as a "constant" for 16,32,64 to restrict the values.